packages/lib/file-api-driver-local.ts

```diff
@@ -13,11 +13,39 @@ const { basicDelta } = require('./file-api');
 // check that it is indeed the problem, check log-database.txt of both clients, search for the note ID, and most likely both notes
 // will have been modified at the same exact second at some point. If not, it's another bug that needs to be investigated.
 
-class FileApiDriverLocal {
-	fsErrorToJsError_(error, path = null) {
+// this is the format that fs-driver-rn would return
+type StdStatResult = {
+	birthtime: Date; // Created date
+	mtime: Date; // Last modified date
+	isDirectory: ()=> boolean; // Is the file a directory?
+	path: string;
+	size: number; // Size in bytes
+};
+
+type MetaData = {
```

Comment:
That's too vague. Maybe give it no type for now (but not "any"), as I don't know what to suggest

Reply:
im not sure what you are referring to ? 
MetaData type you mean ?
If you mean that, well it's name was already decided by `metadataFromStat_` method, so why not use it ?
maybe `StatMetaData` ? 
my opinion is to have this type rather than inlining it